tests/ltp/patches/fix_recvmsg_recvmsg01.patch

```diff
@@ -69,27 +73,39 @@ index 13bcaa4e0..b5ea895f4 100644
  /* 7 */
  	{
  	PF_INET, SOCK_STREAM, 0, iov, -1, (void *)buf, sizeof(buf),
-@@ -246,6 +249,7 @@ int main(int argc, char *argv[])
+@@ -245,7 +248,7 @@ int main(int argc, char *argv[])
+ 	tst_exit();
  }
  
- pid_t pid;
-+pthread_t tid;
+-pid_t pid;
++pthread_t tid = NULL;
  char tmpsunpath[1024];
  
  void setup(void)
-@@ -268,8 +272,9 @@ void setup(void)
+@@ -263,13 +266,19 @@ void setup(void)
+ 
+ 	signal(SIGPIPE, SIG_IGN);
+ 
+-	pid = start_server(&sin1, &sun1);
```

Comment:
please check if sin1 and sun1 are global so is it needed to be passed in as params

Reply:
@ashokchandran, it will avoid usage of & in start_server function. I feel there is no issue if we pass address of these global variable. Please let me know your input.

Reply:
ok that will keep changes minimal. Approved.